testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -269,17 +269,17 @@ public void testCreateTableWithNotNullColumn(String mode)
         }
     }
 
-    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS}, dataProvider = "supportedColumnMappingForDmlDataProvider")
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_91, PROFILE_SPECIFIC_TESTS})
     @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
-    public void testCreatePartitionTableAsSelect(String mode)
+    public void testColumnMappingModeIdCreatePartitionTableAsSelect()
     {
         String tableName = "test_dl_create_partition_table_as_select_" + randomNameSuffix();
 
         onTrino().executeQuery("" +
                 "CREATE TABLE delta.default." + tableName + " " +
                 "WITH (" +
                 "location = 's3://" + bucketName + "/databricks-compatibility-test-" + tableName + "'," +
-                "column_mapping_mode = '" + mode + "'" +
+                "column_mapping_mode = 'id'" +
```

Comment:
The name column mapping mode is more important than id in Delta/Databricks. The id mapping is basically for compatibility with Iceberg. I feel running only with id mapping doesn't make sense.